src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -778,6 +802,7 @@ class HttpGoogleServicesDAO(
   def toGoogleGroupName(groupName: RawlsGroupName) = s"GROUP_${groupName.value}@${appsDomain}"
 
   def adminGroupName = s"${groupsPrefix}-ADMINS@${appsDomain}"
+  def curatorGroupName = s"${groupsPrefix}-CURATORS@${appsDomain}"
```

Comment:
when is this group created? Should we create it automatically if it does not exist?